music_assistant/controllers/streams.py

```diff
@@ -293,9 +303,16 @@ async def setup(self, config: CoreConfig) -> None:
             self.publish_ip,
             self.publish_port,
         )
+        bind_port_value = self.publish_port
```

Comment:
bind port != publish port

Reply:
The penny has now dropped that the config entry only will allow an INT so this is actually simple.